### PR TITLE
 Set section length dynamically

### PR DESCRIPTION
**Pull request by wanshot**
Hi  tell-k
In the case of long model verbose, syntax error of reStructuredText will result.

I set the section length dynamically.

However, I used wcwidth for this fix, but if you do not want to rely on external libraries, reject it

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -6,6 +6,7 @@ deps =
     coverage
     mock
     pbr<1.4
+    wcwidth
```
Comment (tell-k) [inline tox.ini]: This line isn't needed.  Because `wcwidth` exists in install_requires of setup.py.  you should write only the testing library here.

**Review by tell-k — CHANGES_REQUESTED** [review]
@wanshot 

First of all thanks for your nice PR.

I have a question for you.
Why shold we need `wcwdith`?

I do not think that an exact character width is necessary. By converting to bytes, you can get a length longer than the exact character width. That's enough.

```python
# py3
from django.utils.encoding import force_bytes
from wcwidth import wcswidth

print(len(force_bytes('日本語'))) # => 9 (This is enough length).
print(wcswidth('日本語'))         # => 6
```

In other words, it can be a simpler implementation.
Please try this implementaion and modfied your PR.

https://gist.github.com/tell-k/a6b7a2bfb15952fa898c9dcda95efba7

If this is not good, please report again.
Thx for your help :)

**Comment by wanshot** [comment]
Thanks for the reply:satisfied:

Certainly precise number of characters was not necessary.

The suggested modification is better than what I thought. I will refer to it...
I fix it and PR again！

**Comment by tell-k** [comment]
> I fix it and PR again！

yeah!
i'm look forward to your next PR. 

thx.

**Comment by wanshot** [comment]
I will be glad if you look fixed at your convenience!

**Comment by tell-k** [comment]
@wanshot Sorry too late my reply.  Thank you for your help.

**Comment by tell-k** [comment]
I've just released v1.9.0 https://pypi.python.org/pypi/django-modelsdoc/0.1.9

**Comment by wanshot** [comment]
@tell-k  thank you!! :tada:

**Pull request merged**